config/config.go

```diff
@@ -610,22 +610,14 @@ func handleDeprecatedConfig() {
 	deprecatedMap := param.GetDeprecated()
 	for deprecated, replacement := range deprecatedMap {
 		if viper.IsSet(deprecated) {
-			if len(replacement) == 1 {
-				if replacement[0] == "none" {
-					log.Warningf("Deprecated configuration key %s is set. This is being removed in future release", deprecated)
-				} else {
-					log.Warningf("Deprecated configuration key %s is set. Please migrate to use %s instead", deprecated, replacement[0])
-					log.Warningf("Will attempt to use the value of %s as default for %s", deprecated, replacement[0])
-					value := viper.Get(deprecated)
-					viper.SetDefault(replacement[0], value)
-				}
+			if replacement[0] == "none" {
+				log.Warningf("Deprecated configuration key %s is set. This is being removed in future release", deprecated)
 			} else {
-				log.Warningf("Deprecated configuration key %s is set. This is being replaced by %s instead", deprecated, replacement)
-				log.Warningf("Setting default values of '%s' to the value of %s.", replacement, deprecated)
-
-				value := viper.Get(deprecated)
 				for _, rep := range replacement {
+					log.Warningf("The configuration key '%s' is deprecated. Please use '%s' instead.", deprecated, rep)
+					value := viper.Get(deprecated)
```

Comment:
I think I'd still like this log message to mention the expected behavior where we attempt to map deprecated keys to to their replaced values. Can you restore the second warning or lump it into a single warning?